test/ICARECSVClient.test.js

```diff
@@ -20,6 +20,7 @@ const testConfig = {
       },
     },
   ],
+  commonExtractorArgs: {},
```

Comment:
DP - I'm guessing that we added this because the ICARECSVClient was bricking when commonExtractorArgs is null? I think the change we should make in that case is to make the `commonExtractorArgs.implementation` check more robust; we can make the original check `(commonExtractorArgs && commonExtractorArgs.implementation)`, which will short-circuit if the `commonExtractorArgs` obj doesn't exist and avoid an error.